index.html

```diff
@@ -197,12 +197,12 @@ <h1>Skills</h1>
       <div class="motivation">
         <p>I'm always<br> interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
       </div>
-      <div class="form">
-        <input type="text" placeholder="Full Name">
-        <input type="text" placeholder="Email Address">
-        <textarea name="text" id="#" cols="37" rows="7" placeholder="Enter Your Text"></textarea>
-        <button>Git in Touch</button>
-      </div>
+      <form class="form" action="https://formspree.io/f/mpzkevow" method="POST">
```

Comment:
Let's style the form as similar as possible to the template provided in Figma by making these two changes:
  - [ ] Let's increase the weight of the font to match the template
  - [ ] Let's also center the button in the form, instead of the button taking the full width of the page. We could also increase the weight of the font in the button.
  - [ ] Finally, let's style the text and the placeholders of the form as close to the template as we can.
  
 Yours:
![image](https://user-images.githubusercontent.com/79114233/139949805-c4add6fd-0d93-40df-86d4-f33580293043.png)

Template:
![image](https://user-images.githubusercontent.com/79114233/139949835-f4a29fc7-1144-46f2-802c-3ca5640882a7.png)